var/spack/repos/builtin/packages/foonathan-memory/package.py

```diff
@@ -13,4 +13,5 @@ class FoonathanMemory(CMakePackage):
     homepage = "https://memory.foonathan.net/"
     url = "https://github.com/foonathan/memory/archive/v0.7.tar.gz"
 
+    version("0.7-3", sha256="4203d15db22a94a3978eeb1afb59a37d35c57c0f148733f0f1a53a6281cb74dd")
```

Comment:
This should be in a separate PR since it's unrelated

Reply:
Already upgraded, see #38879